tests/autotest1.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+# copyright 2022
+
+import requests
+import time
+import sys
+
+print("starting...")
+try:
+    response = requests.get("https://db.drunkbatya.com", headers = {'host': 'db.dunkbatya.com'}, timeout = 1)
```

Comment:
This line is unreadable, replace `{'host': 'db.dunkbatya.com'}` to `{'host': 'WRONG_HOST'}` or etc.